contrib/ansible/openshift/register.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+default_tools_interval=$1
+ose_master_interval=$2
+ose_node_interval=$3
+file="/tmp/file"
+tool_dir=/var/lib/pbench-agent/tools-default
```

Comment:
So this is an assumption we should document which is that this only works with the `default` pbench tools group.